src/drivers/azure-key-vault.ts

```diff
@@ -107,9 +107,9 @@ export default defineDriver((opts: AzureKeyVaultOptions) => {
 });
 
 const base64Map: { [key: string]: string } = {
-  "=": "e",
-  "+": "p",
-  "/": "s",
+  "=": "-e-",
+  "+": "-p-",
+  "/": "-s-",
```

Comment:
Would it makes sense / be possible that we use url encoding? (`=` ~> `%3D`)

Reply:
Unfortunately not, as Key Vault only supports dashes besides alphanumeric characters. This replacement is only necessary because of the potential characters that could result in a base64 encoding. B64 was the only encoding I found that is available without adding an additional dependency and without requiring a custom implementation of an encoding scheme. As dashes are the only allowed characters and there were too many edge cases with replacing the special characters of b64 with alphanumeric characters or only one dash, this was the least overhead solution.
I can add an additional test of course to validate the encoding.